deeppavlov/deep.py

```diff
@@ -17,6 +17,7 @@
 import argparse
 
 from deeppavlov.core.commands.train import train_evaluate_model_from_config
+from pipeline_manager.pipeline_manager import PipelineManager
```

Comment:
move `pipeline_manager` inside the deeppavlov module.

Reply:
Fixed